src/test/java/calculator/Calculator.java

```diff
@@ -0,0 +1,51 @@
+package calculator;
+
+import java.util.ArrayList;
+
+public class Calculator {
+
+    private static final String DEFAULT_DELIM = " ";
+
+    public long eval(String s) {
+        return eval(s, DEFAULT_DELIM);
+    }
+
+    public long eval(String s, String delim) {
+        if (s == null) {
```

Comment:
```suggestion
        if (Objects.isNull(s)) {
```
Objects# 을 알아보고 활용해보세요 !

Reply:
isNull 이 더 좋을 이유가 있을까요?

Reply:
```java
if (s == null) {
}

if (Objects.isNull(s)) {
}

if (s != null) {
}

if (Objects.nonNull(s)) {
}
```
개인적으로는 가독성 측면에서 좋다고 생각하여 활용하고 있습니다. 
위와 같이 Objects 클래스를 활용하여 ==, !=와 같은 Null 체크를 대신하고 있네요 😄